content/renderers.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework.renderers import JSONRenderer
+
+class PostsJSONRenderer(JSONRenderer):
```

Comment:
Okay, what exactly are we extending here that the base JSONResponse class can't do? You should answer what edge cases this satisfies in a class comment.